### PR TITLE
Naga Belly Color

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Naga use hair color on their belly.

# How to test:
Tell if it looks good.

### CODE REVIEW

File: common/genes/wc_gene_body_naga_test.txt [hunk 1]
```diff
@@ -45,6 +45,36 @@ morph_genes = {
 					}
 					priority = 1
 				}
+				decal = {
```
Comment (kaldrskegg) [inline common/genes/wc_gene_body_naga_test.txt:48]: Can we do this for scales as well on the back and back of head?

**Review by kaldrskegg — CHANGES_REQUESTED** [review]
Wondering if we can do this for the back scales also? But I would wait a bit to approve because the masks might have to change as I am working on the diffuse right now.

File: common/genes/wc_gene_body_naga_test.txt (hunk 1 above)
Comment (zumbak04) [reply common/genes/wc_gene_body_naga_test.txt:48]: > Can we do this for scales as well on the back and back of head?

Which areas?

Comment (kaldrskegg) [reply common/genes/wc_gene_body_naga_test.txt:48]: May be unecessary. In WoW you end up with 4 colors: belly, skin, scales and webbing.
image
Like here, webbing pink, scales green, belly red, skin orange. If its not doable then its fine.

**Review by kaldrskegg — APPROVED** [review]
LGTM

**Pull request merged**